cachi2/core/package_managers/yarn_classic/main.py

```diff
@@ -1,18 +1,60 @@
 from cachi2.core.models.input import Request
 from cachi2.core.models.output import Component, EnvironmentVariable, RequestOutput
+from cachi2.core.package_managers.yarn.utils import run_yarn_cmd
```

Comment:
**Commit message:**

nitpick: I'd also probably reference #626 from here for completeness and have a record in the git history directly even though you have it linked in the PR.